packages/transport/src/modules/messages/MessageController.ts

```diff
@@ -333,19 +348,16 @@ export class MessageController extends TransportController {
         const addressToRelationshipId: Record<string, ICoreId> = {};
 
         for (const recipient of parsedParams.recipients) {
-            const relationship = await this.relationships.getActiveRelationshipToIdentity(CoreAddress.from(recipient));
-            if (!relationship) {
-                throw TransportCoreErrors.messages.missingOrInactiveRelationship(recipient.toString());
-            }
+            const relationship = await this.relationships.getRelationshipToIdentity(CoreAddress.from(recipient));
 
-            const signature = await this.secrets.sign(relationship.relationshipSecretId, plaintextBuffer);
+            const signature = await this.secrets.sign(relationship!.relationshipSecretId, plaintextBuffer);
```

Comment:
Usually we shouldn't use `!` in productive coding. I'm not sure if it okay here, since we do know that the relationship must exists. Does anyone else have an opinion on this?